### ISSUE TITLE
changed "More Options" in Block Toolbar for Embed blocks

### ISSUE THREAD

**Issue by bph** [WordPress/gutenberg#27339]
Prior to WordPress 5.6 the Embed blocks had the following **More Options**

- Hide Block Settings
- Copy
- Duplicate
- Insert Before
- Insert After
- move to
- Edit as HTML
- Add to Reusable blocks
- Group
- Remove Block

![more options](https://user-images.githubusercontent.com/39980/100468364-44151680-30a2-11eb-9fef-b8c9be8e2845.png)

The new version is missing a few of the options: 
- Hide Block Settings
- Copy
- Duplicate
- Insert Before
- Insert After
- move to
- Group
- Remove Block

![Screen Shot 2020-11-27 at 11 14 23 AM](https://user-images.githubusercontent.com/39980/100468454-73c41e80-30a2-11eb-9a4b-fe568deba773.png)

These options are missing
- Edit as HTML
- Add to Reusable blocks

My question:  What was the reasoning behind it these changes? Is there an alternative way to access these features? 

The reason for my inquiry is: On the Documentation team for Block Editor end user, we decided to make More Options a separate page and not have those options repeated on every single block documentation page. Until now, the More Options were uniform. If this those two options were deliberately omitted, we would need to be a bit more specific on the changes to the menu for Embed blocks

**Comment by ntsekouras** [comment]
Hey @bph - thanks for reporting this!

This was introduced in the PR to make `embeds` block variation: https://github.com/WordPress/gutenberg/pull/24090/.
It was a mistake of mine that has slipped..

I've created a new PR to add the support for `html` and `reusable` back here: https://github.com/WordPress/gutenberg/pull/27733

FYI there is still an open issue for `embeds` in front-end: https://github.com/WordPress/gutenberg/issues/14608.

Regarding the documentation these two features/options are available through `supports` option.
Reference: https://developer.wordpress.org/block-editor/developers/block-api/block-supports/

**Comment by bph** [comment]
Thank you, Nik @ntsekouras for letting us know. 

Once the PR is merged, would it also be marked to be added to a 5.6.1 WordPress release? It would be great if we wouldn't have to wait until March for this to be consistent again. This might be a @youknowriad question, though. 

We were just ready to take the "More Options" menu out of the block documentation on many pages and link to one page for it, and it would be helpful if we wouldn't have to keep track to this for the Embed between two versions. We can, of course, if it's not feasible to port this back to core before WP 5.7

**Comment by ntsekouras** [comment]
I'll add the `backport` label in the PR to be checked 👍
